### ISSUE TITLE
Accented uppercase letters on source list

### ISSUE THREAD

**Issue by bxl-forever** [gplv2/urbis-validate#1]
The tool is great and useful but if the source list contains mistakes, it will introduce mistakes in the OSM database.
This is clearly the case for all streets with an accented uppercase letter (usually the French name).
Example : rue de l’Écuyer, rue de l’Évêque, rue des Éburons, etc.
The official name—as shown on street signs—uses an É, not an E.  But the list in "straten_brussel.pdf" uses E everywhere.  Probably that the people who created the list were unaware of proper spelling rules.
Please use judgement and caution when applying changes; this validator is great for spotting obvious mistakes but beware that when a name does not match your list, it does not automatically mean that it was a mistake or that existing OSM data should be changed at all cost.

**Comment by gplv2** [comment]
Indeed, I agree, it's a source data issue.  Plus a misunderstanding of the French rules on accents.  Thanks for sending some links.  I will correct this in the code and scan Brussels again.  rfr. http://osm.org/go/0EoTKzdc--?layers=D&changeset=42524971

**Comment by gplv2** [comment]
Hi bxl-forever ,   this issue wasn't lost on me so I stopped making changes,  I would love to correct any mistakes made with the accents but I could use a database with streetnames of brussels in a format that I can digitally parse to compare with and so far I could not find any.  If you can help me to get my hands on a decent dataset with perfect spelling I would be in your debt.  Thank you

**Comment by bxl-forever** [comment]
I am afraid that such a list does not exist yet.

Former research was quite disappointing so far.  Some municipalities have created their own lists but usually they reflect the same mistakes (http://www.auderghem.be/liste-des-rues-dauderghem // same "E" problem such as "Ecoliers") and or even introduce new ones (space after "l’" is incorrect, so we have to edit their list prior to use).  (https://www.bruxelles.be/sites/default/files/bxl/Pentagone%20Est2014.pdf  // almost unusable because not only the names in French are reversed but also the FR and NL column are sorted independently, so we cannot match them easily).

We can be proud that OSM will soon be the most reliable source of information around!  … but we’ll have to do it ourselves.

I will review the existing list and send you the entries I think are incorrect.  I hope it won’t take too much time to complete and bear with me if we need multiple versions, even if I will try to find all of them in the first run.

**Comment by gplv2** [comment]
Hi Yves, thank you so much for your help.  I started to have the impression that a list like that didn't exists.  It's so hard to fix this without reliable sources.  The PDF I found , I kind of hoped it was a good one, I still believe it's rather good, but then we had the accent problem, real bummer.  Maybe I can setup up a webpage to help gather this data.   I'll think about it... would be very cool if we(OSM) own the only accurate street list on BXL.  I wonder how GPS navigation devices handle this, they must be full of errors...

**Comment by bxl-forever** [comment]
I have reviewed the list: here are all possible mistakes in the list.  The full document (*xlsx) provides justification for the changes.  I hope you will be able to parse this file and update the database when appropriate.  Interestingly, despite there were mistakes in the source list, sometimes the current data in OSM is correct.

There are lots of inconsistencies in Brussels street names but this is mostly due to municipalities making incompatible choices: several streets change name when crossing a border (typically the FR name stays the same but NL name changes, e.g. Gentsesteenweg becomes Steenweg op Gent).

[bxl-names-fix_v1.txt](https://github.com/gplv2/urbis-validate/files/1016203/bxl-names-fix_v1.txt)
[bxl-names-fix_v1.xlsx](https://github.com/gplv2/urbis-validate/files/1016204/bxl-names-fix_v1.xlsx)

**Comment by gplv2** [comment]
I noticed this too, in fact the readme of this tool covers cases like this, there are even more difficult ones in the readme.  It's indeed really important to keep an eye on the borders.

**Comment by gplv2** [comment]
Merci pour la liste, I can really work with this one,  I will probably also get a third source as well, It's possible to extract this list from Urbis data.  might not even be too hard.

Glenn
